site/app/templates/officeHoursQueue/QueueStatus.twig

```diff
@@ -189,9 +189,38 @@
     </form>
   </div>
 {% endif %}
+<br>
+<a id="view_queue_occupancy" onclick="toggleQueueOccupancy()"><button class="btn btn-primary">View Queue Occupancy</button></a>
+<a id="close_queue_occupancy" onclick="toggleQueueOccupancy()" style="display: none"><button class="btn btn-secondary">Close Table</button></a>
```

Comment:
Feels a little weird to have the button change color when toggled from "Open" to "Close". Can we use the same color for both?